analyses/org.osate.analysis.architecture/src/org/osate/analysis/architecture/PropertyTotals.java

```diff
@@ -105,10 +105,28 @@ private double calcPrice(ComponentInstance ci) {
 	 * {@link Result}:
 	 * <ul>
 	 *   <li>{@link Result#getSourceReference()}: Set to the {@code ComponentInstance} for this {@code Result}.
-	 *   <li>{@link Result#getValues()}: One {@code RealValue} is created.
+	 *   <li>{@link Result#getValues()}: Four {@code RealValue} are created.
 	 *   <ul>
-	 *     <li>{@link RealValue#getValue()}: The calculated weight of the component.
-	 *     <li>{@link RealValue#getUnit()}: Weight units are in {@code kg}.
+	 *     <li>Index {@code 0} is a {@code RealValue} for the calculated weight.
+	 *     <ul>
+	 *       <li>{@link RealValue#getValue()}: The calculated weight of the component.
```

Comment:
I think it would be helpful to include one or two sentences that describe the difference between weight, gross weight, and net weight.